CDP4CrossViewEditor/CDP4CrossViewEditor.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=OfficeRibbon/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
Why is this needed?

Reply:
Because there is a weird namespace mispattern but it is actually intentional. I guess this makes sure the user is not bothered by the warning.

Reply:
It's deliberate. By example: check namespace warning inside BasicRdlRibbonPart.cs file and also CrossViewEditorRibbonPart.cs